src/controllers/expenses.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+const expencesServices = require('../services/expenses');
+const userServices = require('../services/users');
+
+const getAll = (req, res) => {
+  const searchParams = req.query;
+
+  if (!searchParams) {
+    res.send(expencesServices.getAll());
```

Comment:
Fix all asynchronous operations - you need to wait for results! In your case you return Promise wrapper, not result. This happens across you controllers.